randovania/games/fusion/logic_database/Sector 1 SRX.txt

```diff
@@ -574,12 +574,17 @@ Extra - room_id: [17]
   > Door to Lava Lake Annex
       Any of the following:
           Gravity Suit and Varia Suit
-          Damage Runs (Intermediate) and Lava Damage ≥ 70
+          Damage Runs (Intermediate) and Lava Damage ≥ 60
+          Varia Suit and Damage Runs (Beginner) and Lava Damage ≥ 21
+          Gravity Suit and Damage Runs (Beginner) and Lava Damage ≥ 30
   > Door to Zebesian Tower
```

Comment:
What about jump extends?

Reply:
Getting regular wall jump physics means that jump extends don't help anymore - unspinning will negate the extra horizontal movement the wall jump grants.

Reply:
There isnt any mention of wall jumps in there tho? Unless they also apply to normal jumps

Reply:
Oof, this terminology is confusing. I should have said "kick jump physics", which is any jump that involves kicking off of the wall. In addition to the "wall jump", there is also the kick jumps off of ladders and  from ledge grabs, which use the same physics as a wall jump. In this case, the jump used here is a kick jump by hanging off the cliffside ledge.

Reply:
> the jump used here is a kick jump by hanging off the cliffside ledge.

Then this should probably have beginner movement attached to it IMO.

Reply:
Sure, and added damage values for not getting the jump to the platform